ext/mbstring/mbstring.h

```diff
@@ -109,7 +108,6 @@ ZEND_BEGIN_MODULE_GLOBALS(mbstring)
 	/* Whether an explicit internal_encoding / http_output / http_input encoding was set. */
 	bool internal_encoding_set;
 	bool http_output_set;
-	bool http_input_set;
```

Comment:
Why are the two other fields not removed?

Reply:
Because while `http_input_set` was *only* written from the handler for writes to `mbstring.http_input`, `http_output_set` (for example) is also written by the function `mb_http_output`. Same for `internal_encoding_set`.

Reply:
ACK, we probably should deprecate the functions that affect those fields.